.travis.yml

```diff
@@ -1,49 +1,59 @@
 language: php
-sudo: required
-dist: precise
+sudo: true
```

Comment:
Yeah, just not possible right now with the Mongo server installs:

https://github.com/php-cache/cache/pull/24/files#diff-cd9e89841f228de7b138486546b36872R1